src/components/Admin/DepartmentTable/index.tsx

```diff
@@ -85,7 +85,14 @@ const DepartmentTable = observer((props: Props) => {
                                 onClick={() => adminDepartmentTable.setSortColumn('color')}
                             />
                         </th>
-                        <th>Klassen-Buchstaben</th>
+                        <th>
+                            <Translate
+                                id="components.admin.filiere.class.letters"
```

Comment:
Use english terms for ids and descriptions

Reply:
Of course. An old mistake...
Done.